lib/Zonemaster/CLI.pm

```diff
@@ -552,16 +554,18 @@ sub run {
     }
 
     my ( $domain ) = @{ $self->extra_argv };
+
     if ( not $domain ) {
         die __( "Must give the name of a domain to test.\n" );
     }
 
-    if ( $domain =~ m/\.\./i ) {
-        die __( "The domain name contains consecutive dots.\n" );
-    }
+    ( my $errors, $domain ) = normalize_name( $domain );
 
-    $domain =~ s/\.$// unless $domain eq '.';
-    $domain = $self->to_idn( $domain );
+    if ( scalar @$errors > 0 ) {
+        my $err;
+        $err .= $_->string . "\n" for @$errors;
```

Comment:
Maybe this line could be rewritten to avoid the use of `$_`? It can be difficult for mere mortals to spot where it was assigned.